desktop/appcontainer/ToastSource.js

```diff
@@ -53,13 +56,18 @@ export class ToastSource {
      * If non-default values are needed for a toaster, a different method must be used.
      *
      * @param {string} [position] - see Blueprint Position enum for allowed values.
+     * @params {Element} [containerRef] - see Blueprint 'container' parameter
      */
-    getToaster(position = Position.BOTTOM_RIGHT) {
-        const toasters = this._toasters;
+    getToaster(position, containerRef) {
```

Comment:
Ah, I see, this is a private class so we are just whacking this extraneous default